src/base/base.ts

```diff
@@ -6,20 +6,18 @@ import type {ContractInputGeneric, ContractMode, ContractState} from '../contrac
 export class Base<M extends ContractMode> {
   readonly contract: Contract<ContractState<M>>;
   readonly warp: Warp;
-  readonly contractTxId: string;
```

Comment:
this is removed because we can get the same property from `contract` object (see `get contractTxId()` in `base/sdk.ts`).

Reply:
does not change anything within the code, API is still the same